spec/views/coronavirus_landing_page/components/landing_page/nhs_notice.html.erb_spec.rb

```diff
@@ -0,0 +1,121 @@
+require "spec_helper"
+
+RSpec.describe "nhs notice partial" do
+  context "header" do
+    it "renders the header as an H2" do
+      render(
+        "coronavirus_landing_page/components/landing_page/nhs_notice",
+        {
+          nhs_banner: {
+            header: "Hello world",
+            sections: [],
+          },
+        }.with_indifferent_access,
+      )
+
+      expect(rendered).to have_selector("h2", text: "Hello world")
+    end
+  end
+
+  context "sections" do
+    it "renders an empty DIV when there are no sections" do
+      render(
```

Comment:
These tests look good. I wonder if it might be possible to DRY up this `render` part by moving it into a method and passing in the `sections` value as an argument from each test?

Reply:
Another great spot - thanks 👍 I went with just using `let(:sections)` instead of method as it seems more RSpecy

Reply:
Also, I think it read better when you apply the `-f d` flag

Reply:
nice one! Agreed the let is better.